### DISCUSSION TITLE
Translate/change fan modes/icons

### DISCUSSION THREAD

**Discussion by HGRT846** [PiotrMachowski/lovelace-xiaomi-vacuum-map-card#731] [Q&A] [answered]
Hello, I use valetudo vacuum and Hypfer/Valetudo platform to control it.
The vacuum advertise these fan modes in attributes:
```
fan_speed_list:
  - low
  - medium
  - high
  - max
  - "off"
```

I use russian, and it does not look nice in the card, only the "medium" mode is translated correctly.
![image](https://github.com/PiotrMachowski/lovelace-xiaomi-vacuum-map-card/assets/55946883/de1efd61-833d-433f-a12d-2052edd25d7e)
How do I change icons/translations for these modes? I tried looking in the code but didn`t really succeed. Hope I didnt duplicate the question.

**Comment by PiotrMachowski** [comment, marked as answer]
Can you:
1. Post your config here
2. Click "Generate static config" in card editor, copy yaml code and post it here as well

**Reply by HGRT846** [reply]
Thanks, I already see where I can change it.
